README.md

```diff
@@ -1,25 +1,25 @@
 # Microsoft Workers for Climate Justice
 
-Yesterday, a major collaboration between Microsoft, Chevron and Schlumberger, an oilfield services company, was [announced](https://news.microsoft.com/2019/09/17/schlumberger-chevron-and-microsoft-announce-collaboration-to-accelerate-digital-transformation/). This partnership outlines Microsoft's use of ["a cloud-based artificial intelligence platform to improve digital services in the oil field.”](https://www.chron.com/business/energy/article/Schlumberger-Chevron-and-Microsoft-launch-14445539.php) With this kind of news making headlines just 3 days before the Global Climate Strike, it's no longer possible for us to ignore Microsoft's complicity in the climate crisis. 
+Yesterday, a major collaboration between Microsoft, Chevron and Schlumberger, an oilfield services company, was [announced](https://news.microsoft.com/2019/09/17/schlumberger-chevron-and-microsoft-announce-collaboration-to-accelerate-digital-transformation/). This partnership outlines Microsoft's use of ["a cloud-based artificial intelligence platform to improve digital services in the oil field.”](https://www.chron.com/business/energy/article/Schlumberger-Chevron-and-Microsoft-launch-14445539.php) This news just days before the Global Climate Strike is a timely example of the awful clash between our economic system and survival of the planet.
```

Comment:
Generally speaking, shifting from active voice to passive voice should be avoided. The purpose of documents such as this are not to present data from a neutral perspective, but rather to convince people to take action. Use of passive voice tends to undercut this goal.

Reply:
I was actually more trying to remove the term 'complicit' which I disagreed with. We are all complicit already, by being employed within a capitalist system, by being consumers, having a car, having travelled by air in the past, etc, the list goes on.

Reply:
I think it's important to leave complicit in, see my comment below.